docker/muranoservice/README.md

```diff
@@ -0,0 +1,237 @@
+
+    regionConfigs:
+    Spain2:
+        agentRabbitMq:
+        host: RABBIT_HOST
+        login: RABBIT_LOGIN
+        password: RABBIT_PASSWORD
+
+
+In the networking section, specify the public network and router to be used.
+    [networking]
+    ...
+    external_network = public-ext-net-01 (the public network)
+    router_name = rt2-external (the router name to be created)
+
+
+### Populate the murano database
+
+Use the following command:
+
+    murano-db-manage upgrade
+
+### Import core library.
+It is required to obtain the core library which is in [murano repository](https://github.com/openstack/murano/tree/master/meta/io.murano)
+We need to execute the zip file for that folder and import in murano by:
+
+    murano package-import --exists-action u  --is-public io.murano.zip
+
+For obtaining the files for the murano core library, it is required to obtain the core library from murano Openstack repository plus the Fiware
+specific requirements. An example to do it can be:
+
+    mkdir /murano && cd /murano && git init && git remote add -f origin https://github.com/openstack/murano && git config core.sparseCheckout true && echo "meta/io.murano/" >> .git/info/sparse-checkout && git pull origin stable/newton
+    mkdir /fiware-murano && cd /fiware-murano && git init && git remote add -f origin https://github.com/telefonicaid/fiware-murano  && git config core.sparseCheckout true && echo "docker/meta/io.murano/" >> .git/info/sparse-checkout && git pull origin develop
+    cp -rf /fiware-murano/docker/* /murano
+    echo "  io.murano.resources.FiwareMuranoInstance: resources/FiwareMuranoInstance.yaml" >> /murano/meta/io.murano/manifest.yaml
+
+### Import Murano packages
+
+import a package from a local .zip file, run:
+
+    $ murano package-import /path/to/PACKAGE.zip
+
+where PACKAGE is the name of the package stored on your computer.
+
+For example:
+
+    $ murano package-import /home/downloads/mysql.zip
+    Importing package com.example.databases.MySql
+    +---------------------------------+------+----------------------------+--------------+---------+
+    | ID                              | Name | FQN                        | Author       |Is Public|
+    +---------------------------------+------+----------------------------+--------------+---------+
+    | 83e4038885c248e3a758f8217ff8241f| MySQL| com.example.databases.MySql| Mirantis, Inc|         |
+    +---------------------------------+------+----------------------------+--------------+---------+
+
+Murano fiware packages are in the following [link](https://github.com/telefonicaid/fiware-enablers/tree/develop/murano-apps).
+
+## How to use muranoservice with Docker
+
+There are several options to use FIWARE_MURANO very easily using docker. These are (in order of complexity):
+
+- _"Have everything automatically done for me"_. See Section **1. The Fastest Way** (recommended).
```

Comment:
Where is section 2?

And after it, where is the descripcion of section 2 and section 3?